README.md

```diff
@@ -198,7 +198,7 @@ It's possible to create simple rules for placing new windows. Currently most use
     });
 ```
 
-The `wm_class` of a window can be looked up by clicking <kbd>Super</kbd><kbd>Insert</kbd> and then checking the value of `metaWindow.wm_class` in emacs or looking glass.
+The `wm_class` of a window can be looked up by clicking <kbd>Super</kbd><kbd>Insert</kbd> and then checking the value of `metaWindow.wm_class` in emacs or looking glass, or typing `xprop | grep WM_CLASS` at the command line then clicking on the window.
```

Comment:
It's probably good to mention that this only works on X11 windows.

It might make sense to direct users to the windows tab in looking glass, `Alt-F2 lg`, which gives you an overview of all windows and their app-id and wm-class since that works in wayland too.

Reply:
Thanks.  Qualified it for X11.  I'm not familiar with Looking Glass (or wayland) so will not try to provide instructions for those.